build/pgo/Terminal.PGO.props

```diff
@@ -14,16 +14,16 @@
     <!-- Mandatory.  Name of the NuGet package which will contain PGO databases for consumption by build system. -->
     <PGOPackageName>Microsoft.Internal.Windows.Terminal.PGODatabase</PGOPackageName>
 
-    <!-- Mandatory.  Major version number of the PGO database which should match the version of the product.  This can be hardcoded or obtained from other sources in build system. -->
+    <!-- Mandatory.  Major version number of the PGO database which should match the version of the product.  This can be hard-coded or obtained from other sources in build system. -->

```

Comment:
Most dictionaries distinguish between nouns and verbs ... Typically the noun loses the `-` and the verb / tenses don't.

E.g. `setup` (n). `to set up` (v).

> Dictionary
Definitions from Oxford Languages · Learn more
Search for a word
hard-code
/härd kōd/
verb COMPUTING
past tense: hard-coded; past participle: hard-coded


I'm happy to drop.

Reply:
English isn't my first language so I was a bit surprised about this. I was looking up whether there are any rules to forming compound words in English and the Cambridge dictionary says:
> We usually make compound nouns with a noun + noun, with a verb (or a word made from a verb) + noun, or with an adjective + noun

Since hard-code is a adjective + verb, I suppose the dash is the right choice at all times?

Reply:
Wait... English considers words with a dash as compound words? As a German, I find that rather pathetic. 😏 Rechtsschutzversicherungsgesellschaften. 👻